shodan/__main__.py

```diff
@@ -617,22 +617,23 @@ def stats(limit, facets, filename, query):
 
 
 @main.command()
-@click.option('--color/--no-color', default=True)
+@click.option('--streamer', help='Specify a custom Shodan stream server to use for grabbing data.', default='https://stream.shodan.io', type=str)
 @click.option('--fields', help='List of properties to output.', default='ip_str,port,hostnames,data')
 @click.option('--separator', help='The separator between the properties of the search results.', default='\t')
-@click.option('--limit', help='The number of results you want to download. -1 to download all the data possible.', default=-1, type=int)
 @click.option('--datadir', help='Save the stream data into the specified directory as .json.gz files.', default=None, type=str)
-@click.option('--ports', help='A comma-separated list of ports to grab data on.', default=None, type=str)
-@click.option('--quiet', help='Disable the printing of information to the screen.', is_flag=True)
-@click.option('--timeout', help='Timeout. Should the shodan stream cease to send data, then timeout after <timeout> seconds.', default=0, type=int)
-@click.option('--streamer', help='Specify a custom Shodan stream server to use for grabbing data.', default='https://stream.shodan.io', type=str)
-@click.option('--countries', help='A comma-separated list of countries to grab data on.', default=None, type=str)
 @click.option('--asn', help='A comma-separated list of ASNs to grab data on.', default=None, type=str)
 @click.option('--alert', help='The network alert ID or "all" to subscribe to all network alerts on your account.', default=None, type=str)
+@click.option('--countries', help='A comma-separated list of countries to grab data on.', default=None, type=str)
+@click.option('--custom', help='A space-separated list of filters query to grab data on.', default=None, type=str)
```

Comment:
I propose to use another parameter name than "custom", e.g. "custom-filter"
```suggestion
@click.option('--custom-filter', help='A space-separated list of filters query to grab data on.', default=None, type=str)
```

Reply:
Agree, but I will change it to plural noun instead so `--custom-filters`, since we already have `countries`, or `ports`